Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM php:7.1-apache
+MAINTAINER Fork CMS <info@fork-cms.com>
+
+# Enable Apache mod_rewrite
+RUN a2enmod rewrite
+
+# Install the dependencies and PHP extensions needed
+RUN apt-get update && \
+    apt-get install -y libpng12-dev libjpeg-dev libmcrypt-dev libicu-dev && rm -rf /var/lib/apt/lists/* && \
+    docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr && \
+    docker-php-ext-install gd mysqli opcache mcrypt intl mbstring pdo pdo_mysql zip
+
+# Custom php.ini settings
+COPY docker/php/php.ini /usr/local/etc/php/
+
+# Enable Xdebug
+COPY docker/php/xdebug.ini xdebug.ini
+RUN pecl install xdebug-2.5.0 && \
+        docker-php-ext-enable xdebug && \
+        cat xdebug.ini  >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini
```

Comment:
Shouldn't there be some custom xdebug settings in our php.ini? If I remember correctly, Symfony and twig get over the maximum stack trace size with the default xdebug settings, causing white screens sometimes.